kotlinx.fuzz.examples/kotlinx.cli/build.gradle.kts

```diff
@@ -6,12 +6,13 @@ plugins {
     id("kotlinx.fuzz.gradle")
 }
 
+val libs = extensions.getByType<VersionCatalogsExtension>().named("libs")
+
 dependencies {
     implementation(kotlin("reflect"))
-    implementation(libs.kotlinx.cli)
-    implementation(libs.commons.cli)
+    implementation(libs.findLibrary("kotlinx-cli").get())
```

Comment:
The solution with `findLibrary` is very cool. Unfortunately, it doesn't work on my machine

Reply:
Also, you don't use it for `kotlinx.html`, `kotlinx.io`, `kotlinx.datetime` and `kotlinx.collections.immutable`

Reply:
That is very strange, because it did not work in the original version for me. I will deal with it